uber/custom_tags.py

```diff
@@ -533,6 +533,17 @@ def render(self, context):
             return ', '.join(costs)
 
 
```

Comment:
We need a custom tag because we need to handle several different cases:
- for single day events we want to just say a date like ``September 18``
- for multi-day events in different months, we want to list both months like ``September 30 - October 1``
- for multi-day events in the same month, we want to list the date range like ``September 5-7``